charts/zookeeper/values.yaml

```diff
@@ -26,12 +26,22 @@ config: {}
   # syncLimit: 2
   # quorumListenOnAllIPs: false
 
+storage:
+    ## configure the storage type
+    ## accepted values : persistence/ephemeral
+    ## default option is
+    type: persistence
+
 persistence:
-  storageClassName: standard
-  ## specifying reclaim policy for PersistentVolumes
-  ## accepted values - Delete / Retain
-  reclaimPolicy: Delete
-  volumeSize: 20Gi
+    storageClassName: standard
+    # specifying reclaim policy for PersistentVolumes
+    # accepted values - Delete / Retain
+    reclaimPolicy: Delete
+    volumeSize: 20Gi
+
+emptydirvolumesource: {}
```

Comment:
Shouldn't this be `ephemeral.emptydirvolumesource`?

Reply:
no, so zookeeper.yaml file is taking value from values file, so in case the user wants to give additional things like medium(choice of memory) or size limit he can specify in  **emptydirvolumesource** field and zookeeper.yaml will take it as shown below eg for **medium: {{ .Values.emptydirvolumesource.medium }}**
```  
 emptydirvolumesource:
        medium: {{ .Values.emptydirvolumesource.medium }}
        sizeLimit: {{ .Values.emptydirvolumesource.sizeLimit }}
```

Reply:
yes, you are correct I have changed it.